packages/mobile/src/qrcode/QRGen.tsx

```diff
@@ -64,54 +69,40 @@ export function transformMatrixIntoPath(cellSize: number, matrix: any) {
 }
 
 interface QRProps {
-  value: string
-  size: number
-  color: string
-  backgroundColor: string
-  getRef: any
-  ecl: string
-  onError: any
-}
-
-interface QRState {
-  cellSize?: number | undefined
-  path?: string | undefined
+  value?: string
+  size?: number
+  color?: string
+  backgroundColor?: string
+  svgRef: React.MutableRefObject<SVG>
+  ecl?: QRCodeErrorCorrectionLevel
+  onError?: any
 }
 
 /**
  * A simple component for displaying QR Code using svg
  */
-export default class QRCode extends PureComponent<QRProps, QRState> {
-  static defaultProps = {
-    value: 'This is a QR Code.',
-    size: 100,
-    color: 'black',
-    backgroundColor: 'white',
-    ecl: 'M',
-    onError: undefined,
-  }
+function QRCode({
+  value = 'This is a QR Code.',
```

Comment:
I see this was taken from the old defaultProps but why do we set this?

Reply:
Yes don't know why it was like this, I'll remove this.